docs/longhornctl_check.md

```
@@ -16,4 +16,4 @@ Longhorn checking operations
 * [longhornctl](longhornctl.md)	 - Longhorn commandline interface.
 * [longhornctl check preflight](longhornctl_check_preflight.md)	 - Check Longhorn preflight
 
-###### Auto generated by spf13/cobra on 13-Oct-2024
+###### Auto generated by spf13/cobra on 27-Nov-2024
```

Markdownlint (0.35.0)

Expected: h4; Actual: h6
Heading levels should only increment by one level at a time

(MD001, heading-increment)